### PR TITLE
Sync for Centers

### PR DESCRIPTION
**Pull request by mayank-kgp**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

#716 #711 (Grammatical mistake removed)

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@therajanmaurya @puneetkohli521 Please review it.

centersync

File: mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java [hunk 1]
```diff
@@ -12,5 +18,49 @@
     public static final String NAME = "Mifos";
 
     //Always Increase the Version Number
-    public static final int VERSION = 2;
+    public static final int VERSION = 3;
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java]: @mayank-kgp you don't need to increase the  DB version every time.

File: mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java [hunk 2]
```diff
@@ -12,5 +18,49 @@
...
+
+    @Migration(version = MifosDatabase.VERSION, database = MifosDatabase.class)
+    public static class Migration1 extends AlterTableMigration<LoanAccount> {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java]: @mayank-kgp don't make add migration classes here. Add new package directory  `database` with name `migrationversion2` and add all migration classes there.

**Comment by therajanmaurya** [comment]
@mayank-kgp I did see any code that is syncing centre's groups and groups' clients.

**Comment by mayank-kgp** [comment]
@therajanmaurya I have done the changes and updated the GIF.

**Pull request merged**